ms/MSOper/MSSummary.cc

```diff
@@ -821,7 +821,7 @@ void MSSummary::listAntenna (LogIO& os, Bool verbose) const
         os << "z";
         os << endl;
         vector<MPosition> antPos = _msmd->getAntennaPositions();
-        Bool posIsITRF = antPos[0].type() != MPosition::ITRF;
+        Bool posIsITRF = antPos[0].getRefPtr()->getType() != MPosition::ITRF;
```

Comment:
Why not antPos[0].getRef().getType() as done in MSMetaData.cc above?

Reply:
It's not clear to me when to use which one. I've changed it now.